ngi_pipeline/conductor/classes.py

```diff
@@ -69,3 +111,45 @@ def add_fastq_files(self, fastq):
         else:
             raise TypeError("Fastq files must be passed as a list or a string: " \
                             "got \"{}\"".format(fastq))
+
+@with_ngi_config
+def get_engine_for_bp(project, config=None, config_file_path=None):
+    """returns a analysis engine module for the given project.
+
+    :param NGIProject project: The project to get the engine from.
+    """
+    charon_session = CharonSession()
+    try:
+        best_practice_analysis = charon_session.project_get(project.project_id)["best_practice_analysis"]
+        if not best_practice_analysis:
+            raise KeyError("For once in my life ever can't you just fill in the forms properly")
+    except KeyError:
+        error_msg = ('No best practice analysis specified in Charon for '
+                     'project "{}". Using "whole_genome_reseq"'.format(project))
+        LOG.error(error_msg)
+        best_practice_analysis = "whole_genome_reseq"
```

Comment:
Would be better to have qc has default engine. Connected to this would be better to have qc has a proper engine